src/Modules/UngradedModules/SubModuleOfFreeModule.jl

```diff
@@ -476,10 +476,38 @@ end
 Check if `a` is an element of `M`.
 """
 function in(a::FreeModElem, M::SubModuleOfFreeModule)
+  iszero(a) && return true
+  any(==(a), gens(M)) && return true
```

Comment:
I think this should also work (but did not test it) and it might be a bit clearer?
```suggestion
  a in gens(M) && return true
```